draft-ietf-quic-transport.md

```diff
@@ -3604,10 +3585,10 @@ A receiver that sends only non-ack-eliciting packets, such as ACK frames, might
 not receive an acknowledgement for a long period of time.  This could cause the
 receiver to maintain state for a large number of ACK frames for a long period of
 time, and ACK frames it sends could be unnecessarily large.  In such a case, a
-receiver could bundle a PING or other small ack-eliciting frame occasionally,
+receiver could send a PING or other small ack-eliciting frame occasionally,
 such as once per round trip, to elicit an ACK from the peer.
 
-A receiver MUST NOT bundle an ack-eliciting frame with packets that would
+A receiver MUST NOT send an ack-eliciting frame in all packets that would
```

Comment:
The "all" was removed in a recent change (I didn't look to see which), but changes the meaning.  I think it's needed.

Reply:
I agree.  (Though it was probably me that deleted it.)